.changeset/loud-snakes-melt.md

```diff
@@ -0,0 +1,5 @@
+---
+'@interledger/open-payments': major
```

Comment:
I don't think this constitutes a major change. In fact, I'm not sure this changes anything in the `open-payments` package itself at all.

Reply:
removed